### PR TITLE
CustomJS / CustomCSS - check file permissions

### PR DESCRIPTION
**Pull request by kevinpapst**
- Adjusted CustomCSS and CustomJS
- added file permission check and error message
- extension config cannot be submitted until permission is fixed
- added german translations
- removed CustomCSS/static/style.css which was loaded by default, included a non-sense CSS rule and was not editable in the frontend

Fixes https://github.com/FreshRSS/FreshRSS/issues/1456

### CODE REVIEW

**Comment by kevinpapst** [comment]
Calling out for @Alkarex and the grammar-eagle-eye @Frenzie ;-)

File: xExtension-CustomCSS/extension.php [hunk 1]
```diff
@@ -19,9 +19,16 @@ public function handleConfigureAction() {
 
 		$current_user = Minz_Session::param('currentUser');
 		$filename =  'style.' . $current_user . '.css';
-		$filepath = join_path($this->getPath(), 'static', $filename);
+		$staticPath = join_path($this->getPath(), 'static');
```
Comment (Frenzie) [inline xExtension-CustomCSS/extension.php]: Oh there's a new path thing. On phone atm.

Comment (Frenzie) [reply xExtension-CustomCSS/extension.php]: Found it https://github.com/FreshRSS/FreshRSS/issues/1456#issuecomment-348797338

Comment (kevinpapst) [reply xExtension-CustomCSS/extension.php]: Yes I saw that, but a) its for 1.9 and b) its not usable right now: https://github.com/FreshRSS/FreshRSS/issues/1709

Comment (Frenzie) [reply xExtension-CustomCSS/extension.php]: Alright, cool.

**Comment by kevinpapst** [comment]
ping @Alkarex

**Comment by Alkarex** [comment]
Sorry, I have a tendency to forget things in other repositories (one of the several reasons I would like to move this one under the main FreshRSS repository).
I have done the French translation.

**Pull request merged**